app/assets/stylesheets/overrides.scss

```diff
@@ -212,3 +212,11 @@ table {
 .orange {
   color: $accent;
 }
+
+.break-word {
+   white-space: pre-wrap;      /* CSS3 */
+   white-space: -moz-pre-wrap; /* Firefox */
+   white-space: -pre-wrap;     /* Opera <7 */
+   white-space: -o-pre-wrap;   /* Opera 7 */
+   word-wrap: break-word;      /* IE */
```

Comment:
Use `//` comments everywhere<br>Line should be indented 2 spaces, but was indented 3 spaces